rest-api/src/main/resources/definitions/sms_message.yml

```diff
@@ -0,0 +1,32 @@
+type: object
+description: Represents an SMS message that we sent to a phone number.
+required:
+    - phoneNumber
+    - sentOn
+    - messageBody
+    - messageId
+    - smsType
+    - studyId
+properties:
+    phoneNumber:
+        type: string
+        description: The phone number receiving this message, formatted in E.164 international format.
+    sentOn:
+        type: string
+        format: date-time
+        description: Timestamp in ISO8601 format when we sent the message.
+    messageBody:
+        type: string
+        description: The message content we sent.
+    messageId:
+        type: string
+        description: Message ID, as determined by the SMS provider.
+    smsType:
+        $ref: ./enums/sms_type.yml
+    studyId:
+        type: string
+        description: The study whose behalf we sent the message.
```

Comment:
I would not expose this as we generally don't. We will always send the message in the study the user is associated to. You can always assume it is in the study you are in when you request it. If that's not true, then it's more like a worker API where the userId isn't in the path and it's not part of the participants API. There aren't really admin apis, since an admin account can switch to any study and carry any additional roles that are needed to perform an operation.

Reply:
As per IM discussion, this is used exclusively for integration tests, where we want to verify the SMS message is being tagged with the correct study ID.